keyboards/ergodox_ez/keymaps/rmw/keymap-win.c

```diff
@@ -0,0 +1,195 @@
+enum custom_keycodes {
+  VRSN = SAFE_RANGE
+  ,OS_CSFT = OSM(MOD_LSFT | MOD_LCTL)
+  ,OS_ALT  = OSM(MOD_LALT)
+  ,OS_CTL  = OSM(MOD_LCTL)
+//  ,TD_SFTX = TD(SFTAND)
+  ,KILLMODS // write a little function that unsets all mods?
```

Comment:
Like `clear_mods();`?

Reply:
Yes, just like. That note is probably two years old... I'll go kill it.

Reply:
```suggestion
```